utils/gear-runtime-test-cli/src/command.rs

```diff
@@ -261,7 +261,8 @@ fn run_fixture(test: &'_ sample::Test, fixture: &sample::Fixture) -> ColoredStri
     let mut errors = vec![];
     let mut expected_log = vec![];
 
-    for message in &fixture.messages {
+    let empty = Vec::new();
+    for message in fixture.messages.as_ref().unwrap_or(&empty) {
```

Comment:
```suggestion
    for message in fixture.messages.as_ref().unwrap_or(&Vec::new()) {
```

Reply:
Change this in other places

Reply:
This can not be done, because we cannot return ref to the temprorary object which cannot outlive its context

Reply:
Same as meant above

Reply:
I've already answered